app/src/main/java/org/stepic/droid/model/CoursesDescriptionContainer.kt

```diff
@@ -1,6 +1,28 @@
 package org.stepic.droid.model
 
+import android.os.Parcel
+import android.os.Parcelable
+
 data class CoursesDescriptionContainer(
         var description: String,
         var colors: CollectionDescriptionColors
-)
\ No newline at end of file
+) : Parcelable {
```

Comment:
Please, check Parcelable implementation like [this](https://github.com/StepicOrg/stepik-android/blob/6a3f3a91b2b9733737b31b534a1fe54e1f8fa13a/app/src/test/java/org/stepic/droid/model/code/ParcelableStringListTest.kt#L13-L13)